designsafe/apps/workspace/static/designsafe/apps/workspace/html/application-form.html

```diff
@@ -57,6 +57,14 @@ <h2>Run <em>{{data.app.label}} <small class="version-label">ver. {{data.app.vers
           sf-options="{ supressPropertyTitles: true }"
           ng-submit="!data.submitting && onSubmit(appForm)"></form>
 
+    <div ng-if="!data.up" style="padding-top: 10px">
```

Comment:
Look at moving this alert to be above the app form, at the same level as the data.unavailable alert.